app/src/main/java/com/example/musclenerds/MainActivity.java

```diff
@@ -85,4 +87,85 @@ public boolean onSupportNavigateUp() {
         return NavigationUI.navigateUp(navController, mAppBarConfiguration)
                 || super.onSupportNavigateUp();
     }
+
+    public void getQuote(View view) {
+        Random rand = new Random();
+        AppExecutors.getInstance().diskIO().execute(new Runnable() {
+            @Override
+            public void run() {
+                List<MotivationalQuote> quotes = mDb.quoteDao().getAll(); //get all quotes.
+                int index = rand.nextInt(quotes.size());
+                Snackbar.make(view, "' " + quotes.get(index).getText() + "' --" + quotes.get(index).getAuthor(), Snackbar.LENGTH_LONG)
+                        .setAction("Action", null).show();
+            }
+        });
+    }
+
+    // Following functions are for generating static data in the database.
+    public void generateQuotes() {
+        MotivationalQuote newQuote = new MotivationalQuote("Today I will do what others won’t, so tomorrow I can accomplish what others can’t.", "Jerry Rice");
+        mDb.quoteDao().insert(newQuote); // insert a quote.
+
+        newQuote = new MotivationalQuote("Do something today that your future self will thank you for.", "Unknown");
+        mDb.quoteDao().insert(newQuote); // insert a quote.
+
+        newQuote = new MotivationalQuote("We are what we repeatedly do. Excellence then is not an act but a habit.", "Aristotle");
+        mDb.quoteDao().insert(newQuote); // insert a quote.
+
+        newQuote = new MotivationalQuote("No matter how slow you go, you are still lapping everybody on the couch.", "Unknown");
+        mDb.quoteDao().insert(newQuote); // insert a quote.
+
+        newQuote = new MotivationalQuote("Sweat is fat crying.", "Unknown");
+        mDb.quoteDao().insert(newQuote); // insert a quote.
+    }
+
+    public void generateExercises() {
+        Exercise newExercise = new Exercise("Sit-Up",
+                "Lie down on your back. bend your legs and stabalize your lower bodey. Cross your hands to opposite shoulders, or place them behind your ears without pulling on your neck. Lift your head and shoulder blades from the ground, Exhale as you rise. Lower, returning to your starting point, exhale as you lower. Repeat.",
+                "abdominal muscles",
+                4,
+                5,
+                "none",
+                "none for now");
+        mDb.exerciseDAO().insert(newExercise);
+        newExercise = new Exercise("Push-Up",
+                "Get down on all fours, placing your hands slightly wider than your shoulders. Straighten your arms and legs. Lower your body until your chest nearly touches the floor. Pause, then push yourself back up. Repeat.",
+                "triceps",
+                4,
+                5,
+                "none",
+                "none for now");
+        mDb.exerciseDAO().insert(newExercise);
+    }
```

Comment:
I assume these functions are just hacks for now. That's fine.

But going forward, any ideas on how we could automate this better?